README.md

```diff
@@ -132,6 +132,23 @@ To start contributing to this plugin, review the [Contributing to Pelican][] doc
 [existing issues]: https://github.com/pelican-plugins/show-source/issues
 [Contributing to Pelican]: https://docs.getpelican.com/en/latest/contribute.html
 
+Testing
+-------
+
+The testing is done using pytest (from the clone/checkout directory).
+
+Linux/Mac:
+```shell
+PYTHONPATH=. py.test -vvs pelican/plugins/test/test_show_source.py
```

Comment:
I would have thought that [pytest-pythonpath](https://github.com/pelican-plugins/show-source/blob/b5f265b5dc303b0015b8a21c683dc1f64334b01a/pyproject.toml#L45) would have made the `PYTHONPATH=.` prefix unnecessary. Is that not the case? Also, I think the new canonical entrypoint is `pytest` rather than `py.test`. Last but not least, and potentially rendering everything I said above moot, I suggest documenting the `invoke tests` command here instead. That way, if the way we run the tests were to change, the documentation for running the tests need not be changed.

Reply:
It doesn't look like: without tests will not run.

This updates the README file https://github.com/GiovanH/show-source/pull/2.

Reply:
This is not related to this pull request and should probably get a separate issue and PR.